spec/suma/admin_api/commerce_products_spec.rb

```diff
@@ -111,7 +111,6 @@ def make_item(_i)
       photo_file = File.open("spec/data/images/photo.png", "rb")
       image = Rack::Test::UploadedFile.new(photo_file, "image/png", true)
       product = Suma::Fixtures.product.create
-      product.add_image({uploaded_file: Suma::Fixtures.uploaded_file.create})
```

Comment:
@rgalanakis I don't think this is testing anything, we just test the product updating with the new image.